frontend/src/metabase/account/app/components/AccountHeader/AccountHeader.styled.jsx

```diff
@@ -34,7 +34,14 @@ export const HeaderSection = styled.div`
 `;
 
 export const HeaderTitle = styled.h2`
+  font-size: 1rem;
```

Comment:
Adjust style to match [the design in Figma](https://www.figma.com/file/KzkeIJPNpgu6p99kvvYUoF/Make-SSO-uneditable?node-id=1%3A1435).